cs/src/comm/epoxy-transport/EpoxyListener.cs

```diff
@@ -134,8 +135,10 @@ internal void InvokeOnDisconnected(DisconnectedEventArgs args)
                         connections.Add(connection);
                     }
 
-                    await connection.StartAsync();
-                    logger.Site().Debug("Started server-side connection for {0}", connection.RemoteEndPoint);
+                    logger.Site().Debug("Setup server-side connection for {0}", connection.RemoteEndPoint);
```

Comment:
If TLS is being used, this still blocks accepting the next connection until the previous one has finished the TLS handshake.